governance/policies/archiving.md

```diff
@@ -98,6 +100,16 @@ When it has been decided that a set of package versions (aka "versions") should
         - `["(none)"]` the maintainer will not maintain any version
         - `["1.3"]` the maintainer will maintain the latest  version of "1.3.Z"
         - `["2.(latest)"]` the maintainer will maintain the latest minor version specifically of version "2" of the package
+- `x-maintained`:
+    - Allowed values: `true` and `false`
+    - Meaning:
+        - Expresses that this opam package version is maintained (if `true`) or not (if `false`).
+        - Overwrites the `x-maintenance-intent` field
+        - Useful to abandon pre-releases
```

Comment:
It is not clear to me how this works or what this means. Since the policy document should stand alone, I think expanding this to explain is a good idea.

Reply:
I clarified in df67251905, hope that is understandable.